### DISCUSSION TITLE
Messages getting exported correctly but no message contents

### DISCUSSION THREAD

**Discussion by stevepsunny** [Tyrrrz/DiscordChatExporter#996] [Q&A] [answered]
Basically the title, I'm able to get the message output from DCE and everything looks correct with all the appropriate reactions and stuff but it isn't showing the actual message content/text for some reason. I dont think this is a permission issue since I give it full admin permissions on the servers I've tested it on.

Anyone have any clue whats going on?

**Comment by 96-LB** [comment, marked as answer]
See #969, #976

**Reply by stevepsunny** [reply]
Sorry for wasting your time, just figured it out a second before you sent it whoops.
